storage/allocator.go

```diff
@@ -61,11 +61,12 @@ const (
 // recommended by the allocator.
 type AllocatorAction int
 
+// These are the possible allocator actions.
 const (
-	aaNoop AllocatorAction = iota
-	aaRemove
-	aaAdd
-	aaRemoveDead
+	AANoop AllocatorAction = iota
+	AARemove
+	AAAdd
```

Comment:
I could name them out in full, AllocatorActionAdd, which makes the most sense to me.  But I'd want to change all of them, not just add.